src/test/java/com/ensono/playwright/ValidatorTest.java

```diff
@@ -67,8 +67,8 @@ public void testValuesPresentInListMatchingValues() {
     }
 
     @Test
-    public void testListIsAlphabetical() {
-        List<String> data = Arrays.asList("aaa", "aab", "abb", "baa", "bba", "bbb", "c", "d");
+    public void testListIsLexicographicallyAlphabetical() {
+        List<String> data = Arrays.asList("aAb", "aaa", "aab", "abb", "baa", "bba", "bbb", "c", "cd", "d");
```

Comment:
`"aAb", "aaa", "aab"`

Is this the expected behaviour?

Reply:
I've made the javadoc for this a bit clearer, whereby this is expected behaviour under lexicographic ordering (capital letters take precedence).